PowerFGT/Public/Connection.ps1

```diff
@@ -168,6 +169,16 @@ function Connect-FGT {
         #Add csrf cookie to header (X-CSRFTOKEN)
         $headers = @{"X-CSRFTOKEN" = $cookie_csrf }
 
+        if($iwrResponse.Content -match '/logindisclaimer')
+        {
+            try {
+                Invoke-WebRequest "https://${Server}:${port}/logindisclaimer" -Method "POST" -WebSession $FGT @invokeParams -Body @{confirm = 1} -Headers $headers | Out-Null
```

Comment:
replace -headers by @invokeParams

Reply:
invokeParams doesn't contain $headers at this point, so I avoided updating invokeParams since it'd then get returned by the function and was unsure of the onward implications.

-headers isn't needed by the looks of it, so I'll just remove it the -headers parameter